pkg/api/files.go

```diff
@@ -77,21 +78,14 @@ func (foldersRequest FoldersRequest) GetFolders() ([]Folder, error) {
 		}
 
 		for _, folderObject := range response {
-			// All the folders and files of a module are stored under the "course files" folder.
-			// We do not want to get that folder as we just want the folders and files in that
-			// folder.
-			//
-			// The "course files" folder resembles the 'home directory' of a module.
-			if folderObject.FullName == "course files" {
-				continue
-			}
-
 			folders = append(folders, Folder{
 				Id:           strconv.Itoa(folderObject.Id),
 				Name:         appFile.CleanseFolderFileName(folderObject.Name),
 				Downloadable: !folderObject.HiddenForUser,
 				HasSubFolder: folderObject.FoldersCount > 0,
 				Ancestors:    ancestors,
+				IsRootFolder: folderObject.ParentFolderId == 0 &&
+					folderObject.FullName == "course files",
```

Comment:
Do note that this is potentially dangerous in the future when Canvas decides to change the name of the default folder from `course files` to something else